### DISCUSSION TITLE
Will Vector 0.32.0 use VRL 0.6.0

### DISCUSSION THREAD

**Discussion by ffcactus** [vectordotdev/vector#18043] [General]
We are waiting for a fix in Vector. In fact, it's in VRL.  https://github.com/vectordotdev/vrl/pull/299
The fix is merged into VRL and will be included in VRL 0.6.0. I would like to know if the next release of Vector (0.32.0) will use it.

Thanks.

**Comment by jszwedko** [comment]
Yep!

**Comment by ffcactus** [comment]
thanks
